universal-login-react/src/ui/UFlow/Devices/ConnectedDevicesItem.tsx

```diff
@@ -31,6 +34,14 @@ export const ConnectedDevicesItem = ({devicesAmount, deviceInfo, publicKey, depl
     }
   };
 
+  const onConfirmDeleteClick = async () => {
+    const currentLocation = history.location.pathname;
+    history.push(join(history.location.pathname, '/waitingForDeleteAccount'));
```

Comment:
In my opinion, using relative paths in navigation generally is an antipattern. It's really error prone and absolute paths are easier to reason about. Please pass `devicesBasePath` prop from parent component and use it to build a URL. Here's an example:
https://github.com/UniversalLogin/UniversalLoginSDK/blob/40d649f9a5d96edb6c8480dbca0b1aaac956dcb4/universal-login-react/src/ui/UFlow/Devices/Devices.tsx#L32
>

Reply:
👍